README.md

```diff
@@ -6,8 +6,13 @@
 
 ## Features
 
-...
-
+- create and delete virtual databases
+- create and delete datasources of predefined types
+- create, modify and delete datasource properties
+- convert datasources to a secret or configmap
```

Comment:
_optional enhancement:_ if datasources is plural, then secret and configmap should be also plural